app/Http/Requests/UpdateJobPoster.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use App\Services\Validation\Rules\DepartmentRule;
+use App\Services\Validation\Rules\ProvinceRule;
+use App\Services\Validation\Rules\SecurityClearanceRule;
+use App\Services\Validation\Rules\LanguageRequirementRule;
+use Illuminate\Support\Facades\Config;
+
+class UpdateJobPoster extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        $dateFormat = Config::get('app.api_datetime_format');
+        $dateFormatRule = "date_format:$dateFormat";
+        return [
+            'term_qty' => 'required|numeric',
+            'open_date_time' =>["required", $dateFormatRule],
```

Comment:
This is pretty nitpicky but can we settle on a quotes/double quotes standard for PHP? It's common to use singlequotes everywhere unless you're interpolating variables.

Reply:
You're absolutely right.